docker/jenkins/periodic_db_update.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Needs DEIS_CONTROLLER URL, DEIS_USERNAME, DEIS_PASSWORD,
+# DEIS_PROFILE, and DEIS_APPLICATION environment variables.
+#
+# To set them go to Job -> Configure -> Build Environment -> Inject
+# passwords and Inject env variables
+#
+
```

Comment:
Good catch. I had included this in the shebang (`#!/bin/bash -xe`) originally, but then pasted over it with an adaptation of your excellent commentary from another script. Will update before merge.